src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessThreadCollection.cs

```diff
@@ -32,7 +32,7 @@ public ProcessThreadCollection(ProcessThread[] processThreads)
         /// </devdoc>
         public ProcessThread this[int index]
         {
-            get { return (ProcessThread)InnerList[index]; }
+            get { return (ProcessThread)InnerList[index]!; }
```

Comment:
Not sure if we need to make it nullable instead, `Add(), Insert()` methods allowing null ...

Reply:
Would that be required needed though? cc @stephentoub

Reply:
I think it's reasonable for us to declare that you are not expected to add null ProcessThreads to the collection, even if technically the implementation allows it.